src/core/Form/Combobox/ComboboxItemList/ComboboxItemList.baseStyles.tsx

```diff
@@ -0,0 +1,32 @@
+import { css } from 'styled-components';
+import { defaultThemeTokens as theme } from '../../../theme';
+import { font } from '../../../theme/reset';
+
+export const baseStyles = css`
+  &.fi-combobox-item-list {
+    ${font(theme)('bodyText')}
+    list-style-type: none;
+
+    border-width: 0 1px 1px 1px;
+    border-style: solid;
+    border-color: ${theme.colors.depthDark3};
+    border-bottom-left-radius: 2px;
+    border-bottom-right-radius: 2px;
+
+    overflow-y: auto;
+    overflow-x: hidden;
+    max-height: 260px;
```

Comment:
Is this something that can be / should be possible to override?

Reply:
Good question. 
This height was chosen based on the wanted amount of items to be shown by the design and to give visual hint of the next items in the list.

Haven't tried to override it by the CSS to see if it is already possible. But hopefully users of the library won't need to customise things like these s that the looks between different services/sites are looking same.

Reply:
I hope so too. Just crossed my mind as I was just working on another fix to allow resizing the Modal.